Application/Sources/UI/Views/LiveRadioSquaredCell.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  LiveRadioSquaredCell.swift
+//  PlaySRG
+//
+//  Created by Mustapha Tarek BEN LECHHAB on 27.01.2025.
+//  Copyright © 2025 SRG SSR. All rights reserved.
+//
```

Comment:
```suggestion
//
//  Copyright (c) SRG SSR. All rights reserved.
//
//  License information is available from the LICENSE file.
//
```

thanks @defagos to catch it in an other PR :)